projects/lights_control/test/test_lights_blinker.c

```diff
@@ -0,0 +1,151 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "test_helpers.h"
+#include "unity.h"
+
+#include "lights_blinker.h"
+#include "lights_events.h"
+
+#define TEST_LIGHTS_BLINKER_DURATION_SHORT 300
+#define TEST_LIGHTS_BLINKER_DURATION_LONG 500
+
+void setup_test(void) {
+  interrupt_init();
+  soft_timer_init();
+  event_queue_init();
+}
+
+void teardown_test(void) {}
+
+void test_lights_blinker_init(void) {
+  LightsBlinker blinker = { 0 };
+  lights_blinker_init(&blinker, TEST_LIGHTS_BLINKER_DURATION_SHORT);
+  TEST_ASSERT_EQUAL(blinker.timer_id, SOFT_TIMER_INVALID_TIMER);
+}
+
+void test_lights_blinker_activate(void) {
```

Comment:
You should probably have a test for your potential error case with on/off that you explained to me.